tests/unit/backend/chalice/api_server/__init__.py

```diff
@@ -14,3 +14,25 @@ class BaseAPITest:
     def setUpClass(cls):
         corpora_api_dir = os.path.join(os.environ["CORPORA_HOME"], "backend", "chalice", "api_server")
         cls.app = ChaliceTestHarness(corpora_api_dir)
+        cls.maxDiff = None
+
+    @staticmethod
+    def remove_timestamps(body: dict) -> dict:
```

Comment:
Since this function is exclusively used for testing, I'd actually stick it back into the testing file.

Reply:
It's used by both test_submission and test_project. It's here to it's inherited by both.

Reply:
It's still test code though, right? Could maybe go in an __init__.py in a test directory or something.

Reply:
It currently live in tests/unit/backend/chalice/api_server/__init__.py. Should I move it somewhere else?

Reply:
Ahh sorry, didn't see that it's still in test. Fine with me!